packages/core/contracts/common/interfaces/Multicall2.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.5.0;
+
+contract Multicall2 {
```

Comment:
Is this code inspired from another external contract? If so we should note it in the comments. For example, the `Multicall.sol` contract notes that its inspired by the MakerDAO contract.